docs/tags/common-schema.md

```diff
@@ -183,3 +183,78 @@ Below are tags that are reserved for future use:
 | service.type | Service type e.g. `hadoop` |
 | span.id | Building block of a trace in distributed tracing |
 | trace.id | Building block of a trace in distributed tracing |
+
+## Events Tags
+
+### Fields
+
+When you create an event you can send a ```JSON``` document which consists of multiple
+fields. Each field can contain event main information or metadata. Even though there is
+no strict format of such a ```JSON``` document we recommend to some of fields. An event
+can contain the following set of fields, most of which are optional:
```

Comment:
@fulyauluturk most are optional or all are optional?
What happens if timestamp is not provided?

Reply:
I think when you try to send an event without timestamp it fails.